docs/ruleset_functions/get_baseline_system_types.md

```diff
@@ -0,0 +1,106 @@
+- Declare a list for SYS-10, Heating and Ventilation w/Electric: `SYS-10 = []`  
+- Declare a list for SYS-11.1, Single Zone VAV System with CHW and Electric Heat : `SYS-11.1 = []`  
+- Declare a list for SYS-11.1a, Single Zone VAV System with PCHW and Electric Heat : `SYS-11.1a = []`  
+- Declare a list for SYS-11b, Single Zone VAV System with CHW and PHW : `SYS-11b = []`  
+- Declare a list for SYS-11c, Single Zone VAV System with PCHW and PHW : `SYS-11c = []`  
+- Declare a list for SYS-11.2, Single Zone VAV System with CHW and HW boiler : `SYS-11.2 = []`  
+- Declare a list for SYS-11.2a, Single Zone VAV System with PCHW and HW boiler : `SYS-11.2a = []`  
+- Declare a list for SYS-12, Single Zone Constant Volume System with CHW and HW boiler : `SYS-12 = []`  
+- Declare a list for SYS-12a, PCHW with HW boiler: `SYS-12a = []`  
+- Declare a list for SYS-12b, CHW with PHW: `SYS-12b = []`  
+- Declare a list for SYS-12c, PCHW and PHW: `SYS-12c = []`  
+- Declare a list for SYS-13, Single Zone Constant Volume System with CHW and Electric Heat : `SYS-13 = []`  
+- Declare a list for SYS-13a, PCHW with and Electric Heat: `SYS-13a = []`  
+
+
+- For each HVAC system in the B_RMR: `hvac_b in B_RMR..HeatingVentilationAirConditioningSystem:`   
+    The two lines below allow for improved efficiency of the code. 
+    - Get list of terminal units associated with the hvac system from the dictionary input to the function: `terminal_unit_id_list = dict_of_zones_and_terminal_units_served_by_hvac_sys[hvac_b.id]["Terminal_Unit_List"]`  
+    - Get list of zone ids associated with the hvac system from the dictionary input to the function: `zone_id_list = dict_of_zones_and_terminal_units_served_by_hvac_sys[hvac_b.id]["Zone_List"]` 
+    - Reset HVAC system type found boolean variable to FALSE: `hvac_sys_type_found = FALSE`  
+    - Call system type 1 function which will return a string of either SYS-1, SYS-1a, SYS-1b, SYS-1c, or Not_Sys_1: `sys_1_type = is_baseline_system_1(B_RMR, hvac_b.id,terminal_unit_id_list,zone_id_list)`  
+    - Check if SYS-1, if it is then add to list of SYS-1s: `if sys_1_type == "SYS-1": SYS-1 = SYS-1.append(hvac_b.id)` 
+    - Check elif SYS-1a, if it is then add to list of SYS-1as: `elif sys_1_type == "SYS-1a": SYS-1a = SYS-1a.append(hvac_b.id)`
+    - Check elif SYS-1b, if it is then add to list of SYS-1bs: `elif sys_1_type == "SYS-1b": SYS-1b = SYS-1b.append(hvac_b.id)`
+    - Check elif SYS-1c, if it is then add to list of SYS-1cs: `elif sys_1_type == "SYS-1c": SYS-1c = SYS-1c.append(hvac_b.id)`
+    - Else, do nothing: `Else:`
+    - Check if sys_1_type does not equal Not_Sys_1: `if sys_1_type != "Not_Sys_1": hvac_sys_type_found = TRUE`  
+    - Check if hvac_sys_type_found = FALSE, if it does carry on if it equals TRUE then loop to the next hvac system: `if hvac_sys_type_found == FALSE:` 
+    - PLACEHOLDER, each is_baseline_system_#() function will be called for each HVAC system in the baseline as shown for system 1.
+
```

Comment:
I think this will work. Not approving it because of a placeholder for other system types.